cmd/packer-sdc/internal/renderdocs/docs-partials/packer-plugin-sdk/provisioners/community_provisioners.mdx

```diff
@@ -0,0 +1,9 @@
+### Community Provisioners
```

Comment:
do we want this one here too?

Reply:
Probably not, I just brought it with the folder, removing